template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseMemoryOvercommitThresholdConfigProvider.java

```diff
@@ -0,0 +1,46 @@
+package com.sequenceiq.cloudbreak.cmtemplate.configproviders.hbase;
+
+import static com.sequenceiq.cloudbreak.cmtemplate.CMRepositoryVersionUtil.CLOUDERA_STACK_VERSION_7_2_17;
+import static com.sequenceiq.cloudbreak.cmtemplate.CMRepositoryVersionUtil.isVersionNewerOrEqualThanLimited;
+import static com.sequenceiq.cloudbreak.cmtemplate.configproviders.ConfigUtils.config;
+
+import java.util.List;
+import java.util.Set;
+
+import org.springframework.stereotype.Component;
+
+import com.cloudera.api.swagger.model.ApiClusterTemplateConfig;
+import com.sequenceiq.cloudbreak.cmtemplate.CmHostGroupRoleConfigProvider;
+import com.sequenceiq.cloudbreak.template.TemplatePreparationObject;
+import com.sequenceiq.cloudbreak.template.views.HostgroupView;
+
+@Component
+public class HbaseMemoryOvercommitThresholdConfigProvider implements CmHostGroupRoleConfigProvider {
+
+    private static final String MEMORY_OVERCOMMIT_THRESHOLD = "memory_overcommit_threshold";
+
+    private static final String MEMORY_OVERCOMMIT_THRESHOLD_DEFAULT_VALUE = "0.9";
+
+    @Override
+    public List<ApiClusterTemplateConfig> getRoleConfigs(String roleType,
+        HostgroupView hostGroupView, TemplatePreparationObject source) {
+        String cdpVersion = source.getBlueprintView().getProcessor().getStackVersion() == null ?
+            "" : source.getBlueprintView().getProcessor().getStackVersion();
+        if (isVersionNewerOrEqualThanLimited(cdpVersion, CLOUDERA_STACK_VERSION_7_2_17)) {
+            return List.of(config(MEMORY_OVERCOMMIT_THRESHOLD,
```

Comment:
why we dont do this on CM side?

Reply:
This is a requirement from one of the customers where value of one of the CM configuration parameters needs to be increased for COD under specific circumstances for worker nodes. Since we already have configproviders on CB side to achieve this, example: 

HiveKnoxConfigProvider class 
HdfsVolumeConfigProvider class, 

which implements CmHostGroupRoleConfigProvider interface. Hence, on similar lines, I defined new config for CM so that CB will send this config for the CM template at install phase.